modules/vector-sets/vset_config.c

```diff
@@ -0,0 +1,48 @@
+/* vector set module configuration.
+ *
+ * Copyright (c) 2009-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of (a) the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+#include "vset_config.h"
+
+#define RM_TRY(expr)                                                  \
+  if (expr == REDISMODULE_ERR) {                                      \
+    RedisModule_Log(ctx, "warning", "Could not run " __STRING(expr)); \
+    return REDISMODULE_ERR;                                           \
+  }
+
+VSConfig VSGlobalConfig;
+
+int set_uint8_numeric_config(const char *name, long long val,
+void *privdata, RedisModuleString **err) {
+REDISMODULE_NOT_USED(name);
+REDISMODULE_NOT_USED(err);
+*(uint8_t *)privdata = (uint8_t) val;
+return REDISMODULE_OK;
+}
+
+long long get_uint8_numeric_config(const char *name, void *privdata) {
+REDISMODULE_NOT_USED(name);
+return (long long)(*(uint8_t *)privdata);
+}
+
+int RegisterModuleConfig(RedisModuleCtx *ctx) {
+  RedisModule_Log(ctx, "warning", "config load from config file");
+  // Numeric parameters
+  RM_TRY(
+    RedisModule_RegisterNumericConfig(
+      ctx, "vectorset-hnsw_max_threads", VSET_DEFAULT_MAX_THREADS,
```

Comment:
Traditionally, we shouldn't use `_` for division config name.

```suggestion
      ctx, "vectorset-hnsw-max-threads", VSET_DEFAULT_MAX_THREADS,
```